bluebrain/repo-bluebrain/packages/neurodamus-model/package.py

```diff
@@ -111,7 +111,11 @@ def build_model(self, spec, prefix):
         # NGV must overwrite other mods, even from the specific
         # models, e.g. ProbAMPANMDA
         if spec.satisfies("+ngv"):
-            copy_all("common_latest/common/mod/ngv", "mod")
+            if spec.satisfies("+metabolism"):
+                copy_all_except_for("common_latest/common/mod/ngv", "mod",
+                                    ["ProbAMPANMDA_EMS.mod"])
```

Comment:
because before running this function, we run the `prepare_mods` function in neurodamus-neocortex which has as well the `ProbAMPANMDA_EMS.mod`. The problem arises from the fact that we want this version of the mod file (neocortex/mod/metabolism) and not the one from common/ngv.